.pre-commit-config.yaml

```diff
@@ -7,23 +7,10 @@ repos:
       - id: end-of-file-fixer
       - id: mixed-line-ending
       - id: trailing-whitespace
-  - repo: https://github.com/psf/black
-    rev: 23.10.1
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
```

Comment:
Not critical, but the URL returns `301 Moved Permanently` response status code and redirects to the up-to-date URL: https://github.com/astral-sh/ruff-pre-commit